openmp/libomptarget/include/Shared/PluginAPI.h

```diff
@@ -219,6 +219,9 @@ int32_t __tgt_rtl_initialize_record_replay(int32_t DeviceId, int64_t MemorySize,
                                            void *VAddr, bool isRecord,
                                            bool SaveOutput,
                                            uint64_t &ReqPtrArgOffset);
+
+// Returns true if the device \p DeviceId suggests to use auto zero-copy.
+int32_t __tgt_rtl_use_auto_zero_copy(int32_t DeviceId);
```

Comment:
Unrelated, but we really need a common method to get target specific information. Something I'm planning on working on when we try to do the whole offload thing.

Reply:
agreed.